flight_phase_files/HALO/EUREC4A_HALO_Flight-Segments_20200215.yaml

```diff
@@ -185,13 +186,15 @@ segments:
   irregularities:
   - TTFS starts 63 seconds prior to first dropsonde to get as close to a full circle as possible
   - early circle exit after 306 degrees
-  - Only 7 dropsondes
+  - Only 8 dropsondes
+  - SAM HALO-0215_s44 overlaps with previous segment (HALO-0215_s5) as per plan
```

Comment:
```suggestion
  - SAM sonde HALO-0215_s44 overlaps with previous segment (HALO-0215_s5)
```

Reply:
I also need to change the circle segment ids from an `s` to a `c`.